app/notification.go

```diff
@@ -333,8 +333,8 @@ func sendNotificationEmail(post *model.Post, user *model.User, channel *model.Ch
 	if *utils.Cfg.EmailSettings.EnableEmailBatching {
 		var sendBatched bool
 		if result := <-Srv.Store.Preference().Get(user.Id, model.PREFERENCE_CATEGORY_NOTIFICATIONS, model.PREFERENCE_NAME_EMAIL_INTERVAL); result.Err != nil {
-			// if the call fails, assume it hasn't been set and don't batch notifications for this user
-			sendBatched = false
+			// if the call fails, assume that the interval has not been explicitly set and batch the notifications
+			sendBatched = true
```

Comment:
I like that idea a lot, but I'm hesitant to do it in this case, as this is a fix for the v4.1 release that is supposed to be cut today. I'd rather not introduce any uncertainty in the release in the name of making the code prettier. If it's ok with you, I'll just keep it in mind going forward for the next time my spidey sense is tingling about some gross code.